docs/docs/manual/installing.md

```diff
@@ -190,10 +190,21 @@ brew install --cask openrefine
 
 <TabItem value="linux">
 
-Once you have downloaded the `.tar.gz` file, open a shell, navigate to the folder containing the download, and type:
+Once you have downloaded the `.tar.gz` file, open the command line (e.g., "Terminal") and navigate to the directory where the file is located.
 
+For instance, if the file is in your `Downloads` directory, use the `cd` command to navigate there.
+
+```
+user@machine:~$ cd Downloads/
+```
+Next, extract the `.tar.gz` file. To extract to the same directory, use the following command:
+  
+```
+user@machine:~$ tar xzf openrefine-3.6.2-linux.tar.gz
```

Comment:
We should probably note that the user will need to use the appropriate file name (e.g. if they are using a different version of OR). We don't do this at the moment but I think it would be a good addition